src/spdx3/bump_from_spdx2/package.py

```diff
@@ -57,10 +71,42 @@ def bump_package(
             description=description,
             comment=comment,
             verified_using=integrity_methods,
+            external_references=external_references,
+            external_identifier=external_identifiers,
             originated_by=originated_by_spdx_id,
             package_purpose=package_purpose,
             package_version=package_version,
             download_location=download_location,
             homepage=homepage,
         )
     )
+
+
+external_ref_map = {
+    "cpe22Type": ExternalIdentifierType.CPE22,
+    "cpe23Type": ExternalIdentifierType.CPE23,
+    "advisory": ExternalReferenceType.SECURITY_ADVISORY,
+    "fix": ExternalReferenceType.SECURITY_FIX,
+    "url": None,
+    "swid": ExternalIdentifierType.SWID,
+    "maven-central": None,
+    "npm": None,
+    "nuget": None,
+    "bower": None,
+    "purl": ExternalIdentifierType.PURL,
+    "swh": ExternalIdentifierType.SWHID,
+    "gitoid": ExternalIdentifierType.GITOID,
+}
+
+
+def bump_external_package_ref(spdx2_external_ref: ExternalPackageRef) -> Union[ExternalReference, ExternalIdentifier]:
+    type = spdx2_external_ref.reference_type
+    locator = spdx2_external_ref.locator
+    comment = spdx2_external_ref.comment
+
+    id_or_ref = external_ref_map[type]
```

Comment:
In SPDX 2.3 there's also `OTHER` as a category, where `reference_type` could be any string, this will lead to a `KeyError` here.